pipelinewise/cli/schemas/tap.json

```diff
@@ -232,6 +232,84 @@
         }
       },
       "required": ["type"]
+    },
+    "is_tap_github": {
+      "required": ["type"],
+      "properties": {
+        "type": {
+          "enum": ["tap-github"]
+        }
+      }
+    },
+    "tap_github": {
+      "anyOf": [
+        {
+          "not": {
+            "$ref": "#/definitions/is_tap_github"
+          }
+        },
+        {
+          "required": ["db_conn", "schemas"],
+          "properties": {
+            "db_conn": {
+              "type": "object",
+              "required": ["access_token", "start_date"],
+              "properties": {
+                "access_token": {
+                  "type": "string"
+                },
+                "start_date": {
+                  "type": "string"
+                },
+                "organization": {
+                  "type": "string"
+                },
+                "repos_include": {
+                  "type": "string"
+                },
+                "repos_exclude": {
+                  "type": "string"
+                },
+                "repository": {
+                  "type": "string"
+                },
+                "include_archived": {
+                  "type": "boolean",
+                  "default": false
+                },
+                "include_disabled": {
+                  "type": "boolean",
+                  "default": false
+                },
+                "max_rate_limit_wait_seconds": {
+                  "type": "integer",
+                  "default": 600,
+                  "min": 600,
+                  "max": 3600
+                }
+              },
+              "allOf": [
+                {
+                  "if": {
+                    "const": "repos_include"
+                  },
+                  "then": {
+                    "not": { "required": ["organization"] }
+                  }
+                },
+                {
+                  "if": {
+                    "const": "repository"
+                  },
+                  "then": {
+                    "not": { "required": ["organization"] }
+                  }
+                }
+              ]
```

Comment:
I don't this block is doing anything tbh, I ran it against some scenarios and it passed when it should have failed.
Better to remove it than keep it.